### PR TITLE
i18n(es): Translated `image-service-reference.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `image-service-reference.mdx` to spanish.

### CODE REVIEW

File: src/content/docs/es/reference/image-service-reference.mdx [hunk 1]
```diff
@@ -0,0 +1,241 @@
+---
+title: API de Servicio de Imágenes
+i18nReady: true
+---
+
+Los [activos](/es/guides/assets/) de Astro fueron diseñados para facilitar que cualquier servicio de optimización de imágenes construya un servicio sobre Astro.
```
Comment (Waxer59) [inline src/content/docs/es/reference/image-service-reference.mdx:6]:
```suggestion
Los [assets](/es/guides/assets/) de Astro fueron diseñados para facilitar que cualquier servicio de optimización de imágenes construya un servicio sobre Astro.
```

File: src/content/docs/es/reference/image-service-reference.mdx [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+
+### `validateOptions()`
+
+**Opcional para ambos servicios locales y externos**
+
+`validateOptions(options: ImageTransform, imageServiceConfig: Record<string, any>): ImageTransform`
+
+Este hook te permite validar y aumentar las opciones pasadas por el usuario. Esto es útil para establecer opciones predeterminadas, o para decirle al usuario que un parámetro es obligatorio.
+
+[Ve como `validateOptions()` se usa en los servicios integrados de Astro](https://github.com/withastro/astro/blob/af4bd5e79c0bd662d58aeb016a61950e176b0a26/packages/astro/src/assets/services/service.ts#L106).
```
Comment (Waxer59) [inline src/content/docs/es/reference/image-service-reference.mdx:217]:
```suggestion
[Mira como `validateOptions()` se usa en los servicios integrados de Astro](https://github.com/withastro/astro/blob/af4bd5e79c0bd662d58aeb016a61950e176b0a26/packages/astro/src/assets/services/service.ts#L106).
```

File: src/content/docs/es/reference/image-service-reference.mdx [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+- **Los servicios locales** se encargan de las transformaciones de imágenes directamente en la compilación para sitios estáticos, o en tiempo de ejecución tanto en el modo de desarrollo como en SSR. Estos son a menudo envoltorios alrededor de bibliotecas como Sharp, ImageMagick o Squoosh. En el modo de desarrollo y en SSR, los servicios locales utilizan un punto final de API para realizar la transformación.
+- **Los servicios externos** apuntan a URLs y pueden agregar soporte para servicios como Cloudinary, Vercel o cualquier servidor compatible con [RIAPI](https://github.com/riapi/riapi).
+
+## Construyendo con la API de Servicios de Imágenes
+
+Las definiciones de servicio toman la forma de un objeto exportado por defecto con varios métodos requeridos ("hooks")
+
+Los servicios externos proporcionan un `getURL()` que apunta al `src` de la etiqueta `<img>` de salida.
+
+Los servicios locales proporcionan un método `transform()` para realizar transformaciones en su imagen, y los métodos `getURL()` y `parseURL()` para utilizar un punto final para el modo de desarrollo y SSR.
```
Comment (Waxer59) [inline src/content/docs/es/reference/image-service-reference.mdx:25]:
```suggestion
Los servicios locales proporcionan un método `transform()` para realizar transformaciones en su imagen, y los métodos `getURL()` y `parseURL()` para utilizar un endpoint para el modo de desarrollo y SSR.
```

File: src/content/docs/es/reference/image-service-reference.mdx [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+Un servicio externo apunta a una URL remota para ser usada como el atributo `src` de la etiqueta `<img>`. Esta URL remota es responsable de descargar, transformar y devolver la imagen.
+
+```js
+import type { ExternalImageService } from "astro";
+
+const service: ExternalImageService = {
+  validateOptions(options: ImageTransform, serviceConfig: Record<string, any>) {
+    // Aplica el ancho máximo establecido por el usuario.
+    if (options.width > serviceConfig.maxWidth) {
+      console.warn(`Image width ${options.width} exceeds max width ${serviceConfig.maxWidth}. Falling back to max width.`);
```
Comment (Waxer59) [inline src/content/docs/es/reference/image-service-reference.mdx:40]:
```suggestion
      console.warn(`"El ancho de la imagen ${options.width} excede el ancho máximo ${serviceConfig.maxWidth}. Se utilizará el ancho máximo como alternativa."`);
```

File: src/content/docs/es/reference/image-service-reference.mdx [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+			status: 200,
+			headers: {
+				'Content-Type': mime.getType(format) || ''
+      }
+    }
+  );
+}
+```
+
+[Ve el endpoint integrado](https://github.com/withastro/astro/blob/main/packages/astro/src/assets/image-endpoint.ts) para un ejemplo completo.
```
Comment (Waxer59) [inline src/content/docs/es/reference/image-service-reference.mdx:152]:
```suggestion
[Mira el endpoint integrado](https://github.com/withastro/astro/blob/main/packages/astro/src/assets/image-endpoint.ts) para un ejemplo completo.
```

File: src/content/docs/es/reference/image-service-reference.mdx [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+```js title="astro.config.mjs"
+import { defineConfig } from "astro/config";
+
+export default defineConfig({
+  experimental: {
+    assets: true,
+  },
+  image: {
+    service: {
+      entrypoint: "your-entrypoint", // 'astro/assets/services/squoosh' | 'astro/assets/services/sharp' | string,
```
Comment (Waxer59) [inline src/content/docs/es/reference/image-service-reference.mdx:234]:
```suggestion
      entrypoint: "tu-punto-de-entrada", // 'astro/assets/services/squoosh' | 'astro/assets/services/sharp' | string,
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by Yan-Thomas — APPROVED** [review]
LGTM!

**Pull request merged**